datafusion/core/src/physical_optimizer/aggregate_statistics.rs

```diff
@@ -189,10 +189,9 @@ fn take_optimizable_column_count(
                     ..
                 } = &col_stats[col_expr.index()]
                 {
-                    let expr = format!("COUNT({})", col_expr.name());
                     return Some((
                         ScalarValue::Int64(Some((num_rows - val) as i64)),
-                        expr,
+                        casted_expr.name().to_string(),
```

Comment:
I wonder if we could use https://docs.rs/datafusion/19.0.0/datafusion/prelude/enum.Expr.html#method.display_name instead of `name` here (which might also resolve any discrepancies) 🤔

Reply:
sorry,i did not get a point.

[display_name](https://docs.rs/datafusion/19.0.0/datafusion/prelude/enum.Expr.html#method.display_name) method was define at  Expr::Column(Column) , The Column type is datafusion_common::Column

but casted_expr type is datafusion::physical_plan::expressions::Column, there is not a method call display_name